spec/ParseQuery.FullTextSearch.spec.js

```diff
@@ -110,7 +111,10 @@ describe('Parse.Query Full Text Search testing', () => {
       }
     };
     await expectAsync(invalidQuery()).toBeRejectedWith(
-      new Parse.Error(Parse.Error.INVALID_JSON, 'bad $text: $search, should be object')
+      new Parse.Error(
+        Parse.Error.INVALID_JSON,
+        ErrorMessage.queryValueTypeInvalid('object', '$text: $search,')
```

Comment:
this looks odd, why is a comma needed at the end of the 2nd argument? Maybe we should move that into the error message?

Reply:
I have moved them to the error message.